src/gc/windows/gcenv.windows.cpp

```diff
@@ -89,6 +91,42 @@ DWORD LCM(DWORD u, DWORD v)
 }
 #endif
 
+bool InitLargePagesPrivilege()
+{
+    TOKEN_PRIVILEGES tp;
+    LUID luid;
+    if (!LookupPrivilegeValueW(nullptr, SE_LOCK_MEMORY_NAME, &luid))
+    {
+        return false;
+    }
+
+    tp.PrivilegeCount = 1;
+    tp.Privileges[0].Luid = luid;
+    tp.Privileges[0].Attributes = SE_PRIVILEGE_ENABLED;
+
+    HANDLE token;
+    if (!OpenProcessToken(::GetCurrentProcess(), TOKEN_ADJUST_PRIVILEGES, &token))
+    {
+        return false;
+    }
+
+    auto retVal = AdjustTokenPrivileges(token, FALSE, &tp, 0, nullptr, 0);
+    auto gls = GetLastError();
```

Comment:
Maoni, are you ok with using "auto" in these places? At least in the other parts of runtime, the coding convention is to use auto only for lambdas or in cases where the type is clearly visible (like in the case of `new something()`).